<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewAssetPage</class>
 <widget class="QDialog" name="NewAssetPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QComboBox" name="contractcomboBox"/>
         </item>
         <item>
          <widget class="QComboBox" name="typecomboBox">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="insertPolicy">
            <enum>QComboBox::NoInsert</enum>
           </property>
           <item>
            <property name="text">
             <string>Token</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Unique</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Equity</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Points</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Credits</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QDateTimeEdit" name="expiry">
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLineEdit" name="inputAmount">
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="placeholderText">
            <string extracomment="Input Amount">Input Amount (0.00) CRW</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="outputAmount">
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="placeholderText">
            <string extracomment="Output Amount">Issuing/Output (0.00)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QCheckBox" name="convertcheckBox">
           <property name="text">
            <string>Convertable</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="transfercheckBox">
           <property name="text">
            <string>Transferable</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="restrictedcheckBox">
           <property name="text">
            <string>Restricted</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="limitedcheckBox">
           <property name="text">
            <string>Limited</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="Create">
         <property name="text">
          <string>Create</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
